scaleway/resource_lb_certificate_beta_test.go

```diff
@@ -0,0 +1,84 @@
+package scaleway
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccScalewayLbCertificateBeta(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckScalewayLbBetaDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource scaleway_lb_beta lb01 {
+						name = "test-lb"
+						type = "lb-s"
+					}
+					resource scaleway_lb_certificate_beta cert01 {
+						lb_id = scaleway_lb_beta.lb01.id
+						name = "test-cert"
+					  	letsencrypt {
+							common_name = "${scaleway_lb_beta.lb01.ip_address}.xip.io"
+					  	}
+					}
+				`,
+				//Check: resource.ComposeTestCheckFunc(
+				//	resource.TestCheckResourceAttr("scaleway_lb_backend_beta.cert01", "name", "test-cert"),
+				//),
```

Comment:
Can we uncomment it now?
```suggestion
				Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr("scaleway_lb_backend_beta.cert01", "name", "test-cert"),
				),
```